### PR TITLE
Lts build fix

### PR DESCRIPTION
**Pull request by tmahmood-microsoft**

### CODE REVIEW

File: e2e/test/E2ETests.csproj [hunk 1]
```diff
@@ -10,8 +10,14 @@
     <RootDir>$(MSBuildProjectDirectory)\..\..</RootDir>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <NoWarn>NU1605,NU1902,NU1903</NoWarn>
```
Comment (tmahmood-microsoft) [inline e2e/test/E2ETests.csproj:14]: these are warnings as errors and are regarding older version of newtonsoft being used.
latest version is 13 and we use 12

**Review by andyk-ms — APPROVED** [review]
Please add the comment about the suppression on each suppression.

File: security/tpm/samples/SecurityProviderTpmSimulator/SecurityProviderTpmSimulator.csproj [hunk 2]
```diff
@@ -2,10 +2,14 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
     <RootNamespace>Microsoft.Azure.Devices.Provisioning.Security</RootNamespace>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <NoWarn>NU1605,NU1902,NU1903</NoWarn>
```
Comment (tmahmood-microsoft) [inline security/tpm/samples/SecurityProviderTpmSimulator/SecurityProviderTpmSimulator.csproj:10]: these are warnings as errors and are regarding older version of newtonsoft being used.
latest version is 13 and we use 12

**Pull request merged**